<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellsPropertiesDialog</class>
 <widget class="QDialog" name="CellsPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>226</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cells Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Borders</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="bottomBorderWidthSpinBox">
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="bottomBorderStyleComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Dotted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dashed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Groove</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ridge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inset</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Outset</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="leftBorderWidthSpinBox">
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="leftBorderButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/border_left.png</normaloff>:/images/border_left.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="leftBorderStyleComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Dotted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dashed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Groove</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ridge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inset</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Outset</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="rightBorderStyleComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Dotted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dashed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Groove</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ridge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inset</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Outset</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="topBorderWidthSpinBox">
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="bottomBorderButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/border_bottom.png</normaloff>:/images/border_bottom.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="rightBorderWidthSpinBox">
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="rightBorderButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/border_right.png</normaloff>:/images/border_right.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="topBorderButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/border_top.png</normaloff>:/images/border_top.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="topBorderStyleComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Dotted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dashed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DotDotDash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Groove</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ridge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inset</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Outset</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="leftBorderColorButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="topBorderColorButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="rightBorderColorButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QToolButton" name="bottomBorderColorButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Padding</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="leftPaddingSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="topPaddingSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="rightPaddingSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="bottomPaddingSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="leftPaddingCheckBox">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="topPaddingCheckBox">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="rightPaddingCheckBox">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="bottomPaddingCheckBox">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Background Color</string>
       </property>
       <property name="buddy">
        <cstring>backgroundColorButton</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="backgroundColorButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CellsPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
